### ISSUE TITLE
Need way to send messages to user.

### ISSUE THREAD

**Issue by bgreenawald** [bgreenawald/madgab-online#1]
Since this application stores the game states in memory and not in a database, anytime we need to deploy new code, everyone's game will be wiped. We need a way to send messages to users on when this will occur (scheduled upgrades). A simple solution would be to store this message in an S3 bucket and have the application read it on load. We have two options for this:

1. The backend reads it and passes it to the front. This is not the ideal solution since it puts additional network strain, both read and write, on the backend.
2. Have the front end read it. The only con I can think of for this is that I don't know the technical difficulty of having a React component read this sort of data.

In either case, this message should be only be shown once per session with the user having the option to disable it.

**Comment by JocelynYH** [comment]
@bgreenawald what is data going to look like?  Should I just do a check against resp.message - ex: if resp.message exists, bring up a maintenance message.

**Comment by bgreenawald** [comment]
If you visit the URL specified you can see the example format I used, but we can format it however you think makes the most sense.

**Comment by JocelynYH** [comment]
Suggestion/comment: 
It would be useful to add a boolean to the response back: 
```{
    "upgrade_message": "The site will be undergoing maintenance December 30, 2019 at 3 am EST. All games will be deleted at that time."
}```

THe boolean would be used to either show the message or not.

**Comment by bgreenawald** [comment]
I completely agree, I'll go ahead and add it to the JSON. I also want to get it set up on S3 rather than my personal site so I'll get it set up and add the boolean as well.

**Comment by JocelynYH** [comment]
nice nice 👍

**Comment by bgreenawald** [comment]
The object is available at https://radgab.s3.amazonaws.com/radgab.json

It's open to the world (for reading) and CORS is enabled, so you should be good to read it however fits best with the frontend structure.

**Comment by JocelynYH** [comment]
Amazing ty!  And yeah, CORS should be enabled - I remember having to edit the CORS configurations to be able to hit the web socket.

**Comment by JocelynYH** [comment]
- [ ] Add rendering/loading circle to make call to this API before landing page content is displayed
